<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Windforce X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QScrollBar" name="m_SliderForceX">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>15</height>
                </size>
               </property>
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Windforce Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QScrollBar" name="m_SliderForceY">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>15</height>
                </size>
               </property>
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Windforce Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QScrollBar" name="m_SliderForceZ">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>15</height>
                </size>
               </property>
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="m_HighspeedX">
            <property name="text">
             <string>enable highspeed windforce X</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_HighspeedY">
            <property name="text">
             <string>enable highspeed windforce Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_HighspeedZ">
            <property name="text">
             <string>enable highspeed windforce Z</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Reset windforce</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="m_CheckBox_SinX">
         <property name="text">
          <string>Windforce sin x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_CheckBox_SinY">
         <property name="text">
          <string>Windforce sin y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_CheckBox_SinZ">
         <property name="text">
          <string>Windforce sin z</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Wind</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Tornado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
